.github/workflows/issue-opened.yml

```diff
@@ -3,13 +3,30 @@ name: Issue Opened Triage
 on:
   issues:
     types: [opened]
+  workflow_dispatch:
+
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  issues: write
+  packages: none
+  pull-requests: none
+  repository-projects: none
+  security-events: none
+  statuses: none
```

Comment:
This is also part of security hardening. 

```
 An action can access the GITHUB_TOKEN through the github.token context even if the 
workflow does not explicitly pass the GITHUB_TOKEN to the action. As a good security practice,
 you should always make sure that actions only have the minimum access they require by 
limiting the permissions granted to the GITHUB_TOKEN. For more information, see 
"Permissions for the GITHUB_TOKEN."
 ```
 
 https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token